omnibus/config/software/freetds.rb

```diff
@@ -22,9 +22,9 @@
   command configure_command, env: env, in_msys_bash: true
   make env: env
 
-  # Only `libtdsodbc.so/libtdsodbc.so.0.0.0` are needed for SQLServer integration.
-  # Hence we only need to copy those.
-  copy "src/odbc/.libs/libtdsodbc.so", "#{install_dir}/embedded/lib/libtdsodbc.so"
+  # Only `libtdsodbc.so.0.0.0` is needed for SQLServer integration.
+  # `libtdsodbc.so` and `libtdsodbc.so.0` symlinks are automatically created
+  # Hence we only need to copy this file.
```

Comment:
> `libtdsodbc.so` and `libtdsodbc.so.0` symlinks are automatically created

Interesting. Do you know how they are automatically created ?

Reply:
@KSerrania tried to investigate but I'm not sure about the conclusion